docs/development.md

```diff
@@ -0,0 +1,17 @@
+# Development
+
+## Compile flags
+
+In order to fix Intellisense issues in VSCode and several other IDEs related to missing header files in C++ dependencies such as `ada-url` and `simdutf`, you need to generate a `compile_commands.json` document. Follow the steps below to fix the issue:
+
+1.  Install `gen-compile-commands`
+
+```sh
+cargo install --git https://github.com/mikea/gen-compile-commands`
+```
+
+2.  Generate `compile_commands.json`
+
+```sh
+gen-compile-commands --root . --compile-flags compile_flags.txt --out compile_commands.json --src-dir ./src
```

Comment:
let's add Justfile and give `just compile-commands` instructions?

Reply:
sounds good!